schemas/data/measure.schema.json

```diff
@@ -19,6 +19,11 @@
           "format": "uri",
           "description": "Links to the `Metric` that this measure refers to. The `@type` can be omitted in contexts where it is obvious what metric is being referred to, for instance in the `Metrics` object that is keyed with `Metric` URIs."
         },
+        "xdm:uniqueId": {
```

Comment:
I think this field is duplicating the [`@type`](https://github.com/adobe/xdm/blob/fac76175274eac81b0e47ee2c39f65b680710784/schemas/data/measure.schema.json#L17), because they both point to the Metric that this measure belongs to.

`uniqueId` has following disadvantages, though:
- it's not spelled `uniqueID` as the `CONTRIBUTING.md` guidelines want
- it's just a string, not a URI, which means it cannot be resolved
- it's unclear who governs the uniqueness of these values